README.md

```diff
@@ -6,24 +6,25 @@ This module is the Web Interface part of the Zonemaster project.
 
 ### Installation
 
-Follow the detailed [installation instructions](docs/Installation.md).
+Follow the detailed [installation instructions].
 
 ##### Prerequisites
-Before you install the Zonemaster Web GUI module, you need the
-Zonemaster Engine test framework installed. Please see the
-[Zonemaster Engine installation instructions](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Installation.md)
 
-And also the the zonemaster-backend module installed. Please see the [Zonemaster
-Backend installation](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Installation.md)
+Before you install the Zonemaster Web GUI module, you need the Zonemaster
+Engine test framework installed. Please see the [Zonemaster-Engine
+installation] document.
+
+And also the the zonemaster-backend module installed. Please see the
```

Comment:
The sentence "And also the the zonemaster-backend module installed" need some reworking. Double "the" and then what does it say?

Reply:
Fine.